platform/pom.xml

```diff
@@ -29,17 +29,11 @@
     <packaging>pom</packaging>
 
     <properties>
-        <cxf.version>3.0.4</cxf.version>
         <cxf.jaxb.version>2.2.6</cxf.jaxb.version>
```

Comment:
I got a little uncomfortable with the thought of moving those around when I saw how many of them there were and how they were defined. It's worth addressing, but I think it should be done in another task with time to focus specifically on those properties.